### ISSUE TITLE
Venom E4 add max health bug with Frozen Heart

### ISSUE THREAD

**Issue by nybardic** [ChalkyBrush/roshpit-bug-tracker#662]
Equip Frozen Heart, and then active E4 add max health.  the health will become 100/80100. 
Then, when E4 max health buff ends，venom's health will suddenly become 1/100, because of the calculation method of health in dota2 is to scale down the current health with the max health.
Need to lock the health to 100 when equip frozen heart , calculate by every second I think.

**Comment by nybardic** [comment]
There are also some bugs  same like this. AXE R3 add health also cause this bug. 
And it will also cause E1 add great attributes easily.

**Comment by ChalkyBrush** [comment]
oh, Dragon didn't realize I changed the Health system just for frozen heart. The new buff uses old HP system so it just needs to be adjusted, thanks for pointing this out

**Comment by ChalkyBrush** [comment]
yes, axe R3 also using old method (I missed this for winterblight patch)

![image](https://user-images.githubusercontent.com/5232984/44247350-bbfcf280-a1b1-11e8-8542-a048329d78d5.png)

Will fix this as well, thanks for the report

**Comment by nybardic** [comment]
Thanks Chalky 👍 !   (c_d is AXE R3 lol)

**Comment by redledcat** [comment]
please fix? solunia r1 arcana as well?
